pages/configuration/server.conf.rst

```diff
@@ -393,6 +393,19 @@ HTTP
     * This is a list of patterns separated by ",". The patterns may start or end with a "*" for wildcards.
     * Any host matching one of these patterns will be reached through a direct connection instead of through a proxy.    
 
+.. _config_script_alert:
+
+Script alert notification
+^^^^^^^^^^^^^^^^^^^^^^^^^
+
+* ``web_interface_url = https://graylog.example.com``
```

Comment:
wouldn't it be better if that is by default `http_external_uri`?

With that, this setting is only needed if this should be different from the default that is already configured. (just as Idea)